packages/database/src/core/PersistentConnection.ts

```diff
@@ -76,9 +76,8 @@ export class PersistentConnection extends ServerActions {
   id = PersistentConnection.nextPersistentConnectionId_++;
   private log_ = logWrapper('p:' + this.id + ':');
 
-  /** @private {Object} */
   private interruptReasons_: { [reason: string]: boolean } = {};
-  private listens_: { [path: string]: { [queryId: string]: ListenSpec } } = {};
+  private readonly listens: Map<string, Map<string, ListenSpec>> = new Map();
```

Comment:
Nit: Can you add a comment to explain what the different strings mean? Maybe something easy:

```
 private readonly listens: Map</* path */ string, Map</* queryId */string, ListenSpec>> = new Map();
```

Reply:
Done. Added as JSDoc as well, so that it shows up in IntelliSense.